resources/assets/less/functions.less

```diff
@@ -34,7 +34,11 @@
 }
 
 .default-box-shadow() {
-  box-shadow: 0px 1px 3px fade(#000, 25%);
+  box-shadow: 0px 1px 3px @box-shadow-spread @box-shadow-color;
```

Comment:
what is exactly wrong here (aside from the fact that there are 5 parameters instead of 4)